packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -254,6 +254,9 @@ fi
 %endif
 
 %changelog
+* Mon Oct 28 2019 Shubham Goyal <shugy@amazon.com> - 1.32.1-1
+- Cache Agent version 1.32.1
```

Comment:
should probably mention https://github.com/aws/amazon-ecs-init/pull/260 in change log